lib/normalizePath.js

```diff
@@ -1,15 +1,28 @@
+// @flow
+
 export default function normalizePath(
   rawPath: string,
   workingDirectory: string
 ): string {
   if (!rawPath) {
     return './';
   }
+
   if (rawPath.startsWith(workingDirectory)) {
     return `.${rawPath.slice(workingDirectory.length)}`;
   }
-  if (rawPath.startsWith('.')) {
+
+  if (rawPath.startsWith('/')) {
```

Comment:
I'm not entirely sure if this is what we are supposed to do here, but it matches the previous behavior so I went with it. I think I tried out a version where it passed `rawPath` straight through if it starts with `/`, but that broke something. But I don't recall anymore.